### PR TITLE
Week6_김희숙_복습과제

### PR DESCRIPTION
**Pull request by kimsook**

### CODE REVIEW

**Comment by greenmoonhg** [comment]
희숙님 안녕하세요!
우선 복습과제 피드백이 늦어져 죄송합니다 ㅠㅅㅠ

1.8 
label encoding 을 할 때 조금 더 도움이 되실 것 같은 for 문을 첨부해드려요!

for column in ['BusinessTravel','Department','EducationField','Gender','JobRole','MaritalStatus','OverTime']:
    X[column] = label.fit_transform(X[column])

이번 과제도 꼼꼼하게 풀이해주신 것 같아요!!
수고하셨습니다!! 좋은 방학 보내세요~~~:)
